asserts/pool.go

```diff
@@ -579,7 +768,8 @@ func (p *Pool) unresolvedBookkeeping() {
 	//  * in error
```

Comment:
it's really weird that this is not doing anything with unresolvedSequences ?? missing tests?

Reply:
Yes, missed that, added.